README.md

```diff
@@ -6,7 +6,7 @@ This repository can be used to create brand new BHoM Toolkits :rocket:
 - Start developing! :rocket:
 
 ## Configure development environment
-Learn how to [set up your development environment for BHoM here](https://bhom.xyz/documentation/Contributing/Getting-started-for-developers/). 
+Learn how to [set up your development environment for BHoM here](https://bhom.xyz/documentation/Guides-and-Tutorials/Coding-with-BHoM/). 
 
 ## Implement the Toolkits
-See the instructions in [the documentation page on the BHoM_Toolkit](https://bhom.xyz/documentation/Basics/The-BHoM-Toolkit).
+See the instructions for in our contributing guides on [implementing a new BHoM Toolkit](https://bhom.xyz/documentation/Contributing/Implementing-a-new-Toolkit/).
```

Comment:
Is this missing a word? Or is `for` supposed to be removed? I'm guessing it's meant to be something like:
```suggestion
See the instructions for creating a toolkit in our contributing guides on [implementing a new BHoM Toolkit](https://bhom.xyz/documentation/Contributing/Implementing-a-new-Toolkit/).
```

Though that seems verbose, so maybe:

```suggestion
See the instructions in our contributing guides on [implementing a new BHoM Toolkit](https://bhom.xyz/documentation/Contributing/Implementing-a-new-Toolkit/).
```

Reply:
It is also late so it could just be my eyes! 👀

Reply:
Good catch!! tbh needed a rewrite 😸 
See new commit!